homeassistant/components/onvif/parsers.py

```diff
@@ -376,16 +407,15 @@ async def async_parse_last_clock_sync(uid: str, msg) -> Event:
 
     Topic: tns1:Monitoring/OperatingTime/LastClockSynchronization
     """
+    date_time = datetime_or_zero(msg.Message._value_1.Data.SimpleItem[0].Value)
```

Comment:
This should be inside the `try` block as well.

Reply:
Done